keyboards/hotdox/config.h

```diff
@@ -0,0 +1,85 @@
+#ifndef ERGODOX_HOTDOX_CONFIG_H
+#define ERGODOX_HOTDOX_CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0xE11D
+
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Z.P.Z.O.
+#define PRODUCT         HotDox
+#define DESCRIPTION     QMK keyboard firmware for HotDox
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 14
+
+#define MOUSEKEY_INTERVAL       20
+#define MOUSEKEY_DELAY          0
+#define MOUSEKEY_TIME_TO_MAX    60
+#define MOUSEKEY_MAX_SPEED      7
+#define MOUSEKEY_WHEEL_DELAY    0
+
+#define TAPPING_TOGGLE  1
+
+/* define if matrix has ghost */
+//#define MATRIX_HAS_GHOST
+
+#define TAPPING_TERM    200
+#define IGNORE_MOD_TAP_INTERRUPT // this makes it possible to do rolling combos (zx) with keys that convert to other keys on hold (z becomes ctrl when you hold it, and when this option isn't enabled, z rapidly followed by x actually sends Ctrl-x. That's bad.)
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+    keyboard_report->mods == (MOD_BIT(KC_LCTL) | MOD_BIT(KC_RCTL)) || \
+    keyboard_report->mods == (MOD_BIT(KC_LSFT) | MOD_BIT(KC_RSFT)) \
```

Comment:
Please make the following changes to this piece of code:
* Replace `keyboard_report->mods` with `get_mods()`.
* Replace `(MOD_BIT(KC_LCTL) | MOD_BIT(KC_RCTL))` with `MOD_MASK_CTRL`.
* Replace `(MOD_BIT(KC_LCTL) | MOD_BIT(KC_RCTL))` with `MOD_MASK_SHIFT`.

Reply:
This. 

Otherwise, some of the recent changes cause the `keyboard_report` structure to not be completely accurate.   To ensure that this works, you'd need to make these changes.